frontend/src/scenes/surveys/SurveyCustomization.tsx

```diff
@@ -134,6 +135,43 @@ export function Customization({ appearance, surveyQuestionItem, onAppearanceChan
                         checked={appearance?.shuffleQuestions}
                     />
                 </div>
+                <div className="mt-1">
+                    <LemonField name="survey_popup_delay" className="font-medium">
+                        {({ onChange }) => {
+                            return (
+                                <div className="flex flex-row gap-2 items-center">
+                                    <LemonCheckbox
+                                        checked={!!appearance?.surveyPopupDelay}
+                                        onChange={(checked) => {
+                                            const surveyPopupDelay = checked ? 5 : undefined
+                                            onChange(surveyPopupDelay)
```

Comment:
and then you don't need to call this onChange, since it refers to the `survey_popup_delay` field value. Right now I'd expect that is being added as well to the survey which isn't necessary